.github/workflows/xcode-versions-build.yml

```diff
@@ -0,0 +1,113 @@
+name: Multi-Xcode Build
+
+on:
+  workflow_dispatch:
+  schedule:
+    - cron: '0 1 * * 0-4'
+  pull_request:
+    branches: [ master ]
+    types:
+      - opened
+      - edited
+      - synchronize
+
+jobs:
+  build:
+    if: startsWith(github.head_ref, 'release/') || github.ref_name == 'master'
+    name: "Build App"
+    runs-on: macos-13
+    strategy:
+      max-parallel: 3
+      matrix:
+        variant: [cocoapods, spm]
+        xcode: [14.3.1, 14.2, 14.1]
+    steps:
+      - name: Git - Checkout
+        uses: actions/checkout@v3
+        with:
+          ref: ${{ github.ref }}
+      - name: Select Xcode Version
+        uses: maxim-lobanov/setup-xcode@v1
+        with:
+            xcode-version: ${{ matrix.xcode }}
+      - name: Install SSH key
+        uses: shimataro/ssh-key-action@v2
+        with:
+          key: ${{ secrets.SSH_KEY }}
+          name: id_rsa_github_actions
+          known_hosts: ${{ secrets.KNOWN_HOSTS }}
+      - uses: webfactory/ssh-agent@v0.7.0
+        with:
+          ssh-private-key: ${{ secrets.SSH_KEY }}
+      - uses: ruby/setup-ruby@v1
+        with:
+          ruby-version: "3.2"
+          bundler-cache: true
+      - name: Setup node
+        uses: actions/setup-node@v3
+        with:
+          node-version: 18.3.0
+      - run: npm install --save slack-message-builder
+      - name: Build and archive app via Cocoapods
+        if: ${{ matrix.variant == 'cocoapods' }}
+        run: |
+          bundle exec fastlane build_cocoapods tests:false skip_package_ipa:false
```

Comment:
Is this for the purpose of making the builds faster, or some other reason?

Having it package would validate that the debug app works across Xcode versions too

Reply:
The opposite, this is ensuring we do package at the expense of build times. Note the variable name `skip_package_ipa`.

The Fastlane script previously used `skip_package_ipa=true` as a default to speed up builds. I allowed it to be set from the outside here so we can pick and choose